Some tests fails on python 3.13 when using parallel testing

As the title said, when using python 3.13, the tests in `tests/text_area` fails, despite succeeding on other python version and when not running in parallel.

```bash
# Succeed
poetry run pytest tests -vvv --deselect tests/snapshot_tests/test_snapshots.py

# Fails
poetry run pytest tests -vvv --deselect tests/snapshot_tests/test_snapshots.py -n 12
```
Here are the logs: (add `| tee pytest_[]_python3.13.txt` to the command)
[pytest_parallel_python3.13.txt](https://github.com/user-attachments/files/17982218/pytest_parallel_python3.13.txt)
[pytest_single_python3.13.txt](https://github.com/user-attachments/files/17982221/pytest_single_python3.13.txt)

(Note that `tests/text_area/test_languages.py::test_register_language` and `tests/text_area/test_languages.py::test_register_language_existing_language` fails both time, so we don't care about them, and `tests/snapshot_tests/test_snapshots.py` would just pollute the logs errors)

Wait, I did not pay attention when closing it, but using `-m 'not syntax'` does not solve my issue.
Here is one of the errors I want to solve:
```text
app = TextAreaApp(title='TextAreaApp', classes={'-dark-mode'}, pseudo_classes={'dark', 'focus'})

    async def test_cursor_page_up(app: TextAreaApp):
        """Pageup moves the cursor up 1 page, retaining column index."""
        async with app.run_test() as pilot:
            text_area = app.query_one(TextArea)
            text_area.load_text("XXX\n" * 200)
            text_area.selection = Selection.cursor((100, 1))
            await pilot.press("pageup")
            margin = 2
>           assert text_area.selection == Selection.cursor(
                (100 - app.console.height + 1 + margin, 1)
            )
E           AssertionError: assert Selection(start=(78, 1), end=(78, 1)) == Selection(start=(79, 1), end=(79, 1))
E             
E             Differing attributes:
E             ['start', 'end']
E             
E             Drill down into differing attribute start:
E               start: (78, 1) != (79, 1)
E               At index 0 diff: 78 != 79
E               
E               Full diff:
E                 (
E               -     79,
E               ?      ^
E               +     78,
E               ?      ^
E                     1,
E                 )
E             
E             Drill down into differing attribute end:
E               end: (78, 1) != (79, 1)
E               At index 0 diff: 78 != 79
E               
E               Full diff:
E                 (
E               -     79,
E               ?      ^
E               +     78,
E               ?      ^
E                     1,
E                 )
E             
E             Full diff:
E             - Selection(start=(79, 1), end=(79, 1))
E             ?                   ^            ^
E             + Selection(start=(78, 1), end=(78, 1))
E             ?                   ^            ^

app        = TextAreaApp(title='TextAreaApp', classes={'-dark-mode'}, pseudo_classes={'dark', 'focus'})
margin     = 2
pilot      = <Pilot app=TextAreaApp(title='TextAreaApp', classes={'-dark-mode'}, pseudo_classes={'dark', 'focus'})>
text_area  = TextArea()

tests/text_area/test_selection_bindings.py:270: AssertionError
```

--- a/tests/text_area/test_selection_bindings.py
+++ b/tests/text_area/test_selection_bindings.py
@@ -255,7 +255,7 @@ async def test_cursor_page_down(app: TextAreaApp):
         await pilot.press("pagedown")
         margin = 2
         assert text_area.selection == Selection.cursor(
-            (app.console.height - 1 - margin, 1)
+            (app.size.height - margin, 1)
         )
 
 
@@ -268,7 +268,7 @@ async def test_cursor_page_up(app: TextAreaApp):
         await pilot.press("pageup")
         margin = 2
         assert text_area.selection == Selection.cursor(
-            (100 - app.console.height + 1 + margin, 1)
+            (100 - app.size.height + margin, 1)
         )
 
 

